### ISSUE TITLE
Text Color Is Not Set

### ISSUE THREAD

**Issue by ghost** [chenzhekl/unblock-youku-firefox#2]
[addon/popup/index.css](https://github.com/chenzhekl/unblock-youku-firefox/blob/master/addon/popup/index.css):
```
.btn {
    width: 60px;
    height: 30px;
    border: none;
    margin: 10px;
    background-color: #eeeeee;
}
```
When using a dark desktop theme, if the font color is not specified, the text will be unreadable. (Screenshots 1 and 2)

Screenshot 1 - Text is unreadable (when the cursor is not on a button): 
![colorbug](https://user-images.githubusercontent.com/32787247/31575412-f7207ea4-b0d5-11e7-800a-fc8de49e835f.png)

Screenshot 2 - Text is hardly readable (when the cursor is on a button): 
![colorbug_mouseover](https://user-images.githubusercontent.com/32787247/31575414-fbfed54c-b0d5-11e7-9777-c0365f9fd5c9.png)

Screenshot 3 - Text is readable after patch (when the cursor is not on a button): 
![expected](https://user-images.githubusercontent.com/32787247/31575417-0229677a-b0d6-11e7-9155-5073501d514c.png)

Screenshot 4 - Text is readable after patch (when the cursor is on a button): 
![expected_mouseover](https://user-images.githubusercontent.com/32787247/31575418-06d53c18-b0d6-11e7-96d8-c943a2bab7be.png)

**Comment by chenzhekl** [comment]
What theme and OS are you using? I am unable to reproduce it on macOS.

**Comment by ghost** [comment]
Sorry for the confusion. The "theme" above is referring to the GTK theme on Linux.
Affected themes: Adwaita Dark, Breeze Dark, and possibly other dark GTK themes

You may wish to view the description of [Text Contrast for Dark Themes](https://addons.mozilla.org/en-US/firefox/addon/text-contrast-for-dark-themes/) to learn about the behavior of Firefox when a dark desktop theme (system theme) is used. That addon, however, does not work on addon popup pages or panel pages.

**Comment by chenzhekl** [comment]
Thanks for the quick response! I'll look into it on Monday.

**Comment by chenzhekl** [comment]
PR merged in 0.1.3
